dbms/include/DB/Common/UInt128.h

```diff
@@ -8,6 +8,11 @@
 namespace DB
 {
 
+#if !__clang__
```

Comment:
I would like to see it in more appropriate value, for example, around inner loop in Aggregator.cpp, where is the origin of this warning...